tests/sentry/tasks/test_post_process.py

```diff
@@ -394,7 +395,7 @@ def test_rule_processor_buffer_values(self):
         ), patch("sentry.constants._SENTRY_RULES", MOCK_RULES), patch(
             "sentry.rules.processor.rules", init_registry()
         ) as rules:
-            MockAction = mock.Mock()
+            MockAction = mock.Mock(spec=EventAction)
```

Comment:
When `spec`  is an object, `mock.__class__` should return the class of the spec object, allowing mocks to pass `isinstance` tests.